packages/devtools/src/components/Feedback/Feedback.styles.ts

```diff
@@ -0,0 +1,55 @@
+import { makeStyles, tokens } from '@fluentui/react-components';
+
+const useMessageClasses = makeStyles({
+  feedbackContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: tokens.spacingHorizontalM,
+    padding: '0.25rem 0',
+    marginTop: '0.5rem',
```

Comment:
hmm, any particular reason to use rem instead of pixels for these?

Reply:
rem is more accessibility-friendly - if the user zooms in or changes the default sizes of fonts, for example, the spacing/sizes scale. If I zoomed in with a font size of 16px, there's a risk that the font size won't adjust with the zoom.